src/test/java/com/google/api/codegen/testdata/discoveries/csharp/csharp_sheets.v4.json.baseline

```diff
@@ -197,10 +211,14 @@ namespace SheetsSample
             // The ID of the sheet to copy.
             int sheetId = 0;  // TODO: Update placeholder value.
 
+            // The ID of the spreadsheet to copy the sheet to.
+            string destinationSpreadsheetId = "";  // TODO: Update placeholder value.
+
             // TODO: Assign values to desired properties of `requestBody`:
             Data.CopySheetToAnotherSpreadsheetRequest requestBody = new Data.CopySheetToAnotherSpreadsheetRequest();
+            requestBody.DestinationSpreadsheetId = destinationSpreadsheetId;
 
-            SpreadsheetsResource.SheetsResource.CopyToRequest request = sheetsService.Spreadsheets.Sheets.CopyTo(requestBody, spreadsheetId, sheetId);
+            Spreadsheets.SheetsOperations.CopyTo request = sheetsService.Spreadsheets.Sheets.CopyTo(requestBody, spreadsheetId, sheetId);
```

Comment:
This is wrong, it's a Java rule that got copied over in the overrides file, it was right as it was.

Didn't actually want to start review yet, I put this pull up very prematurely, I'll tag you guys when I'm ready.